src/widgets/BankDetailsFormWidget.php

```diff
@@ -0,0 +1,33 @@
+<?php
+declare(strict_types=1);
+
+namespace hipanel\modules\client\widgets;
+
+use hipanel\base\CrudController;
+use hipanel\modules\client\models\BankDetails;
+use hipanel\modules\client\models\Contact;
+use hipanel\modules\finance\models\Requisite;
+use yii\base\Widget;
+use yii\widgets\ActiveForm;
+
+class BankDetailsFormWidget extends Widget
+{
+    public Requisite|Contact $parentModel;
+    public ActiveForm $form;
+    public CrudController $controller;
+
+    public function run()
+    {
+        $currencies = array_keys($this->controller->getCurrencyTypes());
```

Comment:
I think we should render this widget only for contacts, that represent requisites. @hiqsol what do you think?